draft-meynell-panrg-scion-research-questions.md

```diff
@@ -196,9 +196,34 @@ This draft assumes the reader is familiar with some of the fundamental concepts
 
 ## ISD, AS identity
 
+The SCION protocol specifies 16 bits and 48 bits to identify the ISD and AS.
```

Comment:
```suggestion
The SCION protocol specifies 16 bits and 48 bits to identify the ISD and AS respectively.
```

Reply:
What is the correct terminology here for the 16+48 bit identifier?

Reply:
We use `ISD-AS` in the drafts, let's keep it consistent